smrf/framework/model_framework.py

```diff
@@ -48,6 +48,22 @@
 from smrf.utils.utils import backup_input, date_range, getqotw
 
 
+class SolarParams:
```

Comment:
I added this class to track the solar parameters within a timestep. Any thoughts on where it should live?

Reply:
I could also see this just moving to the `utils`. Namedtuples have more features than you need and I would go with simple.

My new favorite for Python 3.7 and above are data classes for objects that are type aware:
https://docs.python.org/3/library/dataclasses.html

Maybe one day, once SRMF drops 3.6 support

Reply:
`utils` sounds good. Yeah, data classes are my preference for something like this as well. Some day!

Reply:
Dataclasses are neat! Can't quite drop 3.6 support yet but EOL is slated for 12/2021.